README.md

```diff
@@ -1,44 +1,21 @@
-#This project was abandoned because [Moxie Marlinspike said he was not OK with LibreSignal using the Open Whisper Systems servers and the name "Signal"](https://github.com/LibreSignal/LibreSignal/issues/37#issuecomment-217211165).
+#This GitHub repository has been abandoned.
 
-#[There is a FAQ about what to do when LS stops working (because of the version check)](https://github.com/LibreSignal/LibreSignal/wiki/What-to-do-after-LibreSignal-was-abandoned%3F)
+LibreSignal was started in February 2016 so that people using Google-free devices could communicate with people using the official Signal clients. Signal did not support these devices at the time.
 
-In the same thread, Marlinspike said that he would be willing to consider ["a clean, well written, and well tested" pull request that would add WebSocket support to the Android version of Signal](https://github.com/LibreSignal/LibreSignal/issues/37#issuecomment-226646872). This would effectively eliminate Signal's dependency on GCM and thus allow the official Signal app to function on custom Android ROMs that do not include Google Play Services. There is now [a bounty on making this pull request](https://www.bountysource.com/issues/35722527-create-proper-pull-request-to-add-libresignal-s-websocket-support-to-ows-signal), and the person/team making the pull request would also receive whatever the [BitHub](https://whispersystems.org/blog/bithub/) payout would be at the time. Marlinspike expects it to have high battery consumption and an unreliable user experience, but would be fine with it if it comes with a warning and only runs in the absence of Play Services.
+In May 2016, [Open Whisper Systems's stance on federation](https://whispersystems.org/blog/the-ecosystem-is-moving/) meant that LibreSignal users would have had to get their contacts to switch to LibreSignal if the LibreSignal project had set up its own [Signal-Server](https://github.com/WhisperSystems/Signal-Server). When asked if he was OK with LibreSignal using the Open Whisper Systems servers, [Moxie Marlinspike replied](https://github.com/LibreSignal/LibreSignal/issues/37#issuecomment-217211165) that he was not OK with this due to the operational costs of supporting a third party client. Marlinspike [also said](https://github.com/LibreSignal/LibreSignal/issues/37#issuecomment-226646872) that he would be willing to consider "a clean, well written, and well tested" pull request that would add WebSocket support to Signal so that it could function on Google-free devices. A pull request was [eventually submitted](https://github.com/WhisperSystems/Signal-Android/pull/5962), and WebSocket support was added to Signal on [February 20, 2017](https://github.com/WhisperSystems/Signal-Android/commit/1669731329bcc32c84e33035a67a2fc22444c24b).
```

Comment:
I meant to keep it chronological. The comment is from May 5 and blog post from May 10. So first should be the comment and then the blog post.